object/session.go

```diff
@@ -20,66 +20,151 @@ import (
 	"xorm.io/core"
 )
 
+var (
+	casdoorApplication  = "app-built-in"
+	casdoorOrganization = "built-in"
+)
+
 type Session struct {
 	Owner       string `xorm:"varchar(100) notnull pk" json:"owner"`
```

Comment:
You have changed the PK of session table, please handle DB migration well: https://casdoor.org/docs/deployment/db-migration